feathr_project/feathr/constants.py

```diff
@@ -6,3 +6,21 @@
 # 1MB = 1024*1024
 MB_BYTES = 1048576
 
+INPUT_CONTEXT="PASSTHROUGH"
+
+# For use in registry. 
+# For type definition, think it's like a schema of a table. 
+# This version field is mainly to smooth possible future upgrades, 
+# for example, backward incompatible changes should be introduced in v2, to make sure that features registered with v1 schema can still be used
+REGISTRY_TYPEDEF_VERSION="v1"
+
+TYPEDEF_SOURCE=f'feathr_source_{REGISTRY_TYPEDEF_VERSION}'
+TYPEDEF_FEATHR_PROJECT=f'feathr_workspace_{REGISTRY_TYPEDEF_VERSION}'
+TYPEDEF_DERIVED_FEATURE=f'feathr_derived_feature_{REGISTRY_TYPEDEF_VERSION}'
+TYPEDEF_ANCHOR=f'feathr_anchor_{REGISTRY_TYPEDEF_VERSION}'
+TYPEDEF_ANCHOR_FEATURE=f'feathr_anchor_feature_{REGISTRY_TYPEDEF_VERSION}'
+
+TYPEDEF_ARRAY_ANCHOR=f"array<feathr_anchor_{REGISTRY_TYPEDEF_VERSION}>"
+TYPEDEF_ARRAY_DERIVED_FEATURE=f"array<feathr_derived_feature_{REGISTRY_TYPEDEF_VERSION}>"
+TYPEDEF_ARRAY_ANCHOR_FEATURE=f"array<feathr_anchor_feature_{REGISTRY_TYPEDEF_VERSION}>"
```

Comment:
i don't actually this used anywhere. so i am confused why we need this and how it's used.